### PR TITLE
doc: increate update example tree entities

### PR DESCRIPTION
**Pull request by DoIttikorn**
### Description of change
 increate document update tree entities because I confused using create `createDescendantsQueryBuilder` or `createAncestorsQueryBuilder` for updated but we can use simple pattern update tree entities 

### Pull-Request Checklist

- [x] Code is up-to-date with the `master` branch
- [ ] `npm run format` to apply prettier formatting
- [x] `npm run test` passes with this change
- [ ] This pull request links relevant issues as `Fixes #0000`
- [ ] There are new or updated unit tests validating the change
- [x] Documentation has been updated to reflect this change
- [x] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: docs/tree-entities.md [hunk 1]
```diff
@@ -187,9 +188,16 @@ const a112 = new Category()
...
+// UPDATE
+const updateA1 = await dataSource.manager.findOneBy({ id })
+updateA1.name = "updateA1"
+updateA1.parent = a1
+updateA1.children = [a11, a12]
+await dataSource.manager.save(updateA1)
 ```
 
-To load such a tree use `TreeRepository`:
+To load such a tree use `TreeRepository` example `TreeRepository<Entity>`:
```
Comment (pleerock) [inline docs/tree-entities.md:200]: what does it mean example `TreeRepository<Entity>` ?

**Comment by pleerock** [comment]
I think its better to put "update tree" feature description in a separate section. Please feel free to re-open / create new PR with such changes.

File: docs/tree-entities.md (hunk 1 above)
Comment (DoIttikorn) [reply docs/tree-entities.md:200]: oh, I try to example use TreeRepository on typescript. I'm  trying to find  to use  ```TreeRepository``` I think maybe I put example to using

**Comment by DoIttikorn** [comment]
ok

**Pull request closed without merging**